test/lit/passes/table64-lowering.wast

```diff
@@ -67,4 +67,21 @@
   (func $test_table_fill
     (table.fill $t64 (i64.const 0) (ref.null func) (i64.const 10))
   )
+
+  ;; CHECK:      (func $test_table_init
+  ;; CHECK-NEXT:  (table.init $t64 $elem64
+  ;; CHECK-NEXT:   (i32.wrap_i64
+  ;; CHECK-NEXT:    (i64.const 0)
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:   (i32.wrap_i64
+  ;; CHECK-NEXT:    (i64.const 5)
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:   (i32.wrap_i64
+  ;; CHECK-NEXT:    (i64.const 10)
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT: )
+  (func $test_table_init
+    (table.init $t64 $elem64 (i64.const 0) (i64.const 5) (i64.const 10))
```

Comment:
As mentioned in the comments on the validator, this should be invalid because only the first operand should be an i64.